rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParametricTextDocumentService.java

```diff
@@ -357,10 +359,6 @@ private ParametricFileFacts facts(TextDocumentIdentifier doc) {
         return facts(doc.getUri());
     }
 
-    private ParametricFileFacts facts(TextDocumentItem doc) {
```

Comment:
while I get that this one is removed (most likely due to nobody using it). For the next time I have a preference to keep cleanup in separate PRs.

Reply:
I have reverted all changes not related to core purpose of this PR.